frontend/pulse3d/src/pages/uploads.js

```diff
@@ -130,24 +129,18 @@ export default function Uploads() {
   const [pending, setPending] = useState(true);
   const [filterString, setFilterString] = useState("");
   const [filterColumn, setFilterColumn] = useState("");
-  const [ownerWidth, setOwnerWidth] = useState("180px");
-  const [recordingWidth, setRecordingWidth] = useState("300px");
-  const [uploadWidth, setUploadWidth] = useState("250px");
-  const [createdWidth, setCreatedWidth] = useState("250px");
-  const [analyzedWidth, setAnalyzedWidth] = useState("200px");
-
+  const [ownerWidth, setOwnerWidth] = useState("10%");
+  const [recordingWidth, setRecordingWidth] = useState("30%");
+  const [uploadWidth, setUploadWidth] = useState("23%");
+  const [createdWidth, setCreatedWidth] = useState("15%");
+  const [analyzedWidth, setAnalyzedWidth] = useState("15%");
+  const [sortColumn, setSortColumns] = useState("");
   const uploadTableColumns = [
     {
-      width: "30px",
+      width: "3%",
       admin: false,
       cell: (row) => (
-        <Checkbox
-          id={row.id}
-          disabled={row.jobs.filter((job) => job.status !== "finished").length > 0}
-          checked={checkedUploads.includes(row.id)}
-          onChange={handleCheckedUploads}
-          style={{ width: "1px", margin: "0" }}
-        />
+        <Checkbox id={row.id} checked={checkedUploads.includes(row.id)} onChange={handleCheckedUploads} />
```

Comment:
Did we decide to never disable the checkboxes or at least disable then while in pending state?